core/src/components/select/select.tsx

```diff
@@ -183,6 +183,19 @@ export class Select implements ComponentInterface {
    */
   @Prop() selectedText?: string | null;
 
+  /**
+   * The toggle icon to use. Defaults to `chevronExpand` for `ios` mode,
+   * or `caretDownSharp` for `md` mode.
+   */
+  @Prop() toggleIcon?: string;
+
+  /**
+   * The toggle icon to show when the select is open. If defined, the icon
+   * rotation behavior in `md` mode will be disabled. If undefined, `toggleIcon`
+   * will be used for when the select is both open and closed.
+   */
+  @Prop() toggleIconWhenOpen?: string;
```

Comment:
Thoughts on making `toggleIcon` and `toggleIconWhenOpened` named `collapsedIcon` and `expandedIcon`, respectively? `ion-refresher` uses `pullingIcon` and `refreshingSpinner`, and datetime uses `collapsedIcon` and `expandedIcon` internally. It seems like we tend to change the name entirely when the icon is different.

Reply:
There are a couple reasons I went with the current names:
1. `ion-accordion` uses `toggleIcon` for basically the same thing.
2. `toggleIcon` can also be used for when the select is open/expanded, if `toggleIconWhenOpen` isn't specified.

I'm open to your suggestions though, if you feel it's still worth it.

Reply:
cc @brandyscarney any thoughts on this? You're good with naming things.

Maybe we have `toggleIcon` and `expandedIcon`?

Reply:
We talked in Slack and are going with `toggleIcon` and `expandedIcon` 👍